src/app/beer_garden/router.py

```diff
@@ -334,23 +337,20 @@ def forward(operation: Operation):
         raise RoutingRequestException(f"Unknown connection type {connection_type}")
 
 
-def setup_stomp(garden):
-    host_and_ports = [
-        (
-            garden.connection_params.get("stomp_host"),
-            garden.connection_params.get("stomp_port"),
-        )
-    ]
-    conn = stomp.Connection(host_and_ports=host_and_ports)
-    if garden.connection_params.get("stomp_ssl"):
-        if garden.connection_params.get("stomp_ssl").get("use_ssl"):
-            conn.set_ssl(
-                for_hosts=host_and_ports,
-                key_file=garden.connection_params["stomp_ssl"].get("private_key"),
-                cert_file=garden.connection_params["stomp_ssl"].get("cert_file"),
-            )
+def create_stomp_connection(garden: Garden) -> Connection:
+    """Create a stomp connection wrapper for a garden
+
+    Uses the format_connection_params to rip the "stomp_" prefix from the garden's
+    connection params and constructs a stomp connection wrapper from them.
 
-    return conn
+    Args:
+        garden: The garden specifying
+
+    Returns:
+        The created connection wrapper
+
+    """
+    return Connection(**format_connection_params(garden.connection_params))
```

Comment:
The input for `Connection` should have `subscription_destination` set `None`, since the entry_point should handle the listening. Having `subscription_destination` set `None` will setup the connection to only be able to send